### PR TITLE
Configuration Step Header implementation

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Qbd integration
component: configuration-header
https://user-images.githubusercontent.com/55541808/214527380-072abb65-d3be-41d1-826f-704952f4a6fd.mov

File: src/app/shared/components/qbd/configuration/configuration-step-header/configuration-step-header.component.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.step-header {
+    padding: 24px;
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/configuration/configuration-step-header/configuration-step-header.component.scss:2]: replace with tailwindcss

File: src/styles.scss [hunk 2]
```diff
@@ -236,3 +236,26 @@ p {
 .p-toast-message-content {
     @apply tw-block #{!important};
 }
+
+.configuration {
+    &--section {
+      padding-bottom: 100px;
```
Comment (ashwin1111) [inline src/styles.scss:242]: replace stylings

File: src/app/shared/components/qbd/configuration/configuration-step-header/configuration-step-header.component.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.step-header {
+    @apply tw-p-24-px ;
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/configuration/configuration-step-header/configuration-step-header.component.scss:2]: Can write in a single line `@apply tw-p-24-px tw-border-b-gray-200;`

File: src/styles.scss [hunk 4]
```diff
@@ -236,3 +236,26 @@ p {
 .p-toast-message-content {
     @apply tw-block #{!important};
 }
+
+.configuration {
+    &--section {
+      @apply tw-pb-100-px ;
+    }
+  
+    // &--onboarding-section {
```
Comment (ashwin1111) [inline src/styles.scss:245]: remove unused stylings

File: src/styles.scss [hunk 5]
```diff
@@ -236,3 +236,26 @@ p {
...
+    //   margin: 0px 50px;
+    // }
+  
+    // &--post-onboarding-section {
+    //   margin-top: 100px;
+    // }
+  
+    &--contents {
+        box-sizing: border-box;
```
Comment (ashwin1111) [inline src/styles.scss:254]: tailwind styles please

**Pull request merged**